acquisitions/src/main/resources/thunderjet/mod-orders/features/update_fund_in_poline_when_invoice_approved.feature

```diff
@@ -0,0 +1,151 @@
+@parallel=false
+Feature: Should fail updating fund in poLine when related invoice is approved
```

Comment:
Please provide in some place name of jira ticket to connect this test accordingly

Reply:
Added